package/MDAnalysis/units.py

```diff
@@ -36,7 +36,7 @@
 
 .. _table-baseunits:
 
-.. Table:: Base units in MDAnalysis
+.. Table:: Base units in MDAnalysis as encoded in :data:`MDANALYSIS_BASE_UNITS`
```

Comment:
This isn't rendering properly on RTD for me. See the side panel here.

<img width="674" alt="image" src="https://user-images.githubusercontent.com/12460125/189346153-8970d70f-a744-4d75-8230-2b7baae94d21.png">

Reply:
I haven't checked locally (@BFedder please check) but RTD does not always render as nicely as our own. It also tends to generally look different on RTD.

Reply:
See also PR #3775

Reply:
It renders properly locally for me, with the copyright notice at the bottom of the page, not partially occluded on the right